Core/Config.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2012- PPSSPP Project.
+﻿// Copyright (c) 2012- PPSSPP Project.
```

Comment:
Hmm frankly I have no clue, it's probably from editing in Visual Studio 2017. I can remove it manually I guess.